extensions/amp-sidebar/1.0/amp-sidebar.js

```diff
@@ -110,6 +110,7 @@ export class AmpSidebar extends AMP.BaseElement {
     const toolbarElements =
     Array.prototype.slice
       .call(this.element.querySelectorAll('nav[toolbar]'), 0);
+    toolbarElements.reverse();
```

Comment:
Because we are adding each element as the first child of the body.

So if we have toolbars in our html as:

<body>
first 
second
third
...
<body>

The loop goes as follows when sidebar is applied:

**Iteration 1**

<body>
first 
...
<body>

**Iteration 2**

<body>
second
first 
...
<body>

**Iteration 3**

<body>
third
second
first 
...
<body>

Reversing the array does this in the correct order